client/src/views/devices/GreetDeviceModal.vue

```diff
@@ -367,13 +415,82 @@ async function copyLink(): Promise<void> {
   );
 }
 
+async function sendEmail(): Promise<void> {
```

Comment:
You can add a `sendEmail` function to the `DeviceGreet` class. In theory, the token should always be the same as long as it is not consumed, which means you can just do
```typescript
async sendEmail(): Promise<boolean> {
  const clientHandle = getParsecHandle();
  if (clientHandle !== null && !needsMocks()) {
    const result = await libparsec.clientNewDeviceInvitation(clientHandle, true);
    return result.ok;
  } else {
    return true;
  }
}
```